MobileCenter/MobileCenter/Internals/Channel/MSChannelConfiguration.h

```diff
@@ -23,35 +27,34 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  * Interval for flushing the queue.
- *
- * Default: 15.0
  */
 @property(nonatomic, readonly) float flushInterval;
 
 /**
- * Initializes new `MSChannelConfiguration' instance based on given settings.
+ * Initializes a new instance based on given settings.
  *
  * @param groupID The id used by the channel to determine a group of logs.
+ * @param priority The priority of logs being sent by the channel.
  * @param flushInterval The interval after which a new batch will be finished.
  * @param batchSizeLimit The maximum number of logs after which a new batch will be finished.
  * @param pendingBatchesLimit The maximum number of batches that have currently been forwarded to another component.
  *
  * @return a fully configured `MSChannelConfiguration` instance.
  */
 - (instancetype)initWithGroupID:(NSString *)groupID
+                       priority:(MSPriority)priority
                   flushInterval:(float)flushInterval
                  batchSizeLimit:(NSUInteger)batchSizeLimit
             pendingBatchesLimit:(NSUInteger)pendingBatchesLimit;
 
 /**
- * Initializes and configures a predefined `MSChannelConfiguration' instance based on a given priority enum value.
+ * Initializes a new instance with default settings.
  *
- * @param priority the enum value which determines which configurations to us as presets.
- * @param groupID The groupID that will be used to determine a group of logs.
+ * @param groupID The id used by the channel to determine a group of logs.
  *
- * @return a fully configured `MSChannelConfiguration` instance.
+ * @return a fully configured `MSChannelConfiguration` instance with default settings.
  */
-+ (instancetype)configurationForPriority:(MSPriority)priority groupID:(NSString *)groupID;
+- (instancetype)initDefaultChannelWithGroupID:(NSString *)groupID;
```

Comment:
`initDefaultConfigurationWithGroupID` instead of `initDefaultChannelWithGroupID`? Since it initializes a conf but not a channel.